games-arcade/rocksndiamonds/rocksndiamonds-3.3.1.2.recipe

```diff
@@ -56,6 +56,9 @@ BUILD()
 
 INSTALL()
 {
+	addResourcesToBinaries $portDir/additional-files/rocksndiamonds.rdef \
+		$binDir/rocksndiamonds
```

Comment:
$binDir/rocksndiamonds isn't correct here,  it should be:
$appsDir/"Rocks'n'Diamonds"/rocksndiamonds

PS @fbrosson does the iom file need to go into the ADDITIONEL_FILES? as it's not needed for the build ...